src/api/treeTrackerApi.js

```diff
@@ -161,45 +167,37 @@ export default {
       .then(handleResponse)
       .catch(handleError);
   },
-  /*
-   * get species list
+  /**
+   * Earnings Tool
    */
-  getSpecies(abortController) {
-    const query = `${process.env.REACT_APP_API_ROOT}/api/species?filter[order]=name`;
+  getEarnings() {
+    const query = `earnings.json`;
     return fetch(query, {
       method: 'GET',
       headers: {
         'content-type': 'application/json',
         Authorization: session.token,
       },
-      signal: abortController?.signal,
     })
       .then(handleResponse)
       .catch(handleError);
   },
-
-  /**
```

Comment:
We've lost the docstring for `getEarnings()` – was that deliberate?

Reply:
I was going for brevity and consistency, but I can put it back.